revolv/templates/project/project.html

```diff
@@ -10,6 +10,8 @@
     <script type="text/javascript">
         window.PARTIAL_COMPLETENESS = {{ project.partial_completeness_as_js }}
     </script>
+
+    <link href='http://fonts.googleapis.com/css?family=Lato:100,300,400,700,800' rel='stylesheet' type='text/css'>
```

Comment:
Do you need all these font weights? The download for this css file will be larger than needed if we don't use them :(